icu4j/main/tests/core/src/com/ibm/icu/dev/test/message2/Mf2IcuTest.java

```diff
@@ -105,17 +105,17 @@ public void testMessageFormatDateTimeSkeleton() {
         Date date = new GregorianCalendar(2021, Calendar.NOVEMBER, 23, 16, 42, 55).getTime();
 
         doTheRealDateTimeSkeletonTesting(date, "{{$when :datetime skeleton=MMMMd}}",
-                Locale.ENGLISH, "November 23");
+                Locale.forLanguageTag("en"), "November 23");
```

Comment:
optional: No real difference here, why the less concise version?

Reply:
That change was made after @aphillips 's [comment here](https://github.com/unicode-org/icu/pull/2313/files/5a7c52bbd9a8612af77e0a9d350168d46cbda4b4#r1101844314). @mihnita agreed with the reasoning, too.